README.md

```diff
@@ -25,6 +25,8 @@ This repository contains a short, notional example of SQL & R files designed to
 
 *GOOD LUCK!*
   
-## About My Submission: `[Insert your First Name, Last Name]`
-Replace with your own narrative
-
+## About My Submission: `Rebekah McNichol`
+I was rather off my feet with this one. I had brushed up on Pandas and then saw it was R, and of course that was rustier. 
+I am also less experienced with GitHub having mostly played with simple branching in terminals. 
```

Comment:
This is not a bad thing! I encourage a terminal-first approach because it honestly only reinforces good practices in the GUI